openfisca_web_api/scripts/test-perf-tracking.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
```

Comment:
I didn't manage to get results with this scripts, and a one-line shell command can do the job. This is not necessary.